src/admin/decs.d.ts

```diff
@@ -0,0 +1,2 @@
+// express plugin for adminbro doesn't have types yet. This is a workaround.
+declare module '@admin-bro/express';
```

Comment:
only workaround for the moment, they don't have types in the module.

Reply:
Maybe we can use a `// TODO:` or `// FIX: ` to start these comments above in order to have a way of looking tech debt inside the code later ? wdyt?